js/bybit.js

```diff
@@ -5009,6 +5009,105 @@ module.exports = class bybit extends Exchange {
         return this.parseTransfers (transfers, currency, since, limit);
     }
 
+    async borrowMargin (code, amount, symbol = undefined, params = {}) {
```

Comment:
can we add these exceptions?
```
ccxt.base.errors.ExchangeError: bybit {"ret_code":-6029,"ret_msg":"Amount to borrow has exceeded the user\u0027s estimated max. amount to borrow","ext_code":null,"ext_info":null,"result":null}
ccxt.base.errors.ExchangeError: bybit {"ret_code":-6025,"ret_msg":"Amount to borrow cannot be lower than the min. amount to borrow (per transaction)","ext_code":null,"ext_info":null,"result":null}
ccxt.base.errors.ExchangeError: bybit {"ret_code":-1103,"ret_msg":"An unknown parameter was sent.","ext_code":null,"ext_info":null,"result":null}
ccxt.base.errors.ExchangeError: bybit {"ret_code":-6017,"ret_msg":"Repayment amount has exceeded the total liability","ext_code":null,"ext_info":null,"result":null}

```

Reply:
Add them how? I don't understand the edits that need to be made

Reply:
@samgermain, just add it to the exceptions object and map it to the correct Exception, to avoid throwing generic `ExchangeError` 
![image](https://user-images.githubusercontent.com/43336371/183855587-ff859bcb-abe2-4a17-b857-3c10a77ee34b.png)